### PR TITLE
Update the documentation on charts with zero metrics

### LINKED ISSUE #5845: swapio chart is missing on CentOS 7
##### Bug report summary
swapio chart is missing on CentOS 7

##### OS / Environment
```bash
$ uname -a
Linux ***** 3.10.0-957.el7.x86_64 #1 SMP Thu Nov 8 23:39:32 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux

$ cat /etc/redhat-release
CentOS Linux release 7.6.1810 (Core)
```

##### Netdata version (ouput of `netdata -V`)
netdata v1.12.2, v1.13.0 (static binary)

##### Component Name
proc

##### Steps To Reproduce
install netdata on centos 7 and check web UI

##### Expected behavior
system swap should have swapio chart

##### Note

if the system start to use swap, swapio chart become available.

### PR DESCRIPTION
**Pull request by vlvkobal**
##### Summary
By default, Netdata will enable monitoring metrics for disks, memory, and network only when they are not zero. We need the corresponding statements across the documentation.

Closes #5845 

##### Component Name
documentation

### CODE REVIEW

**Comment by ilyam8** [comment]
We need a global option to disable 

> Netdata will enable monitoring metrics for disks, memory, and network only when they are not zero

logic

**Comment by ilyam8** [comment]
i suggest to open an issue @vlvkobal

**Comment by vlvkobal** [comment]
I've opened a new issue #6315

File: collectors/cgroups.plugin/README.md [hunk 1]
```diff
@@ -101,6 +101,10 @@ The whole point for the additional pattern list, is to limit the number of times
 
 The above pattern list is matched against the path of the cgroup. For matched cgroups, netdata calls the script [cgroup-name.sh](cgroup-name.sh.in) to get its name. This script queries `docker`, or applies heuristics to find give a name for the cgroup.
 
+### charts with zero metrics
+
+By default, Netdata will enable monitoring metrics only when they are not zero. If they are constantly zero they are ignored. Metrics that will start having values, after netdata is started, will be detected and charts will be automatically added to the dashboard (a refresh of the dashboard is needed for them to appear though). Set `yes` for a chart instead of `auto` to enable it permanently.
```
Comment (cakrit) [inline collectors/cgroups.plugin/README.md]: Set `yes` where?

Comment (vlvkobal) [reply collectors/cgroups.plugin/README.md]: The "charts with zero metrics" is a subsection of the "configuring netdata for cgroups". But I've added an example to make it more clear.

File: collectors/cgroups.plugin/README.md [hunk 2]
```diff
@@ -103,7 +103,7 @@ The above pattern list is matched against the path of the cgroup. For matched cg
 
 ### charts with zero metrics
 
-By default, Netdata will enable monitoring metrics only when they are not zero. If they are constantly zero they are ignored. Metrics that will start having values, after netdata is started, will be detected and charts will be automatically added to the dashboard (a refresh of the dashboard is needed for them to appear though). Set `yes` for a chart instead of `auto` to enable it permanently. Fox example:
+By default, Netdata will enable monitoring metrics only when they are not zero. If they are constantly zero they are ignored. Metrics that will start having values, after netdata is started, will be detected and charts will be automatically added to the dashboard (a refresh of the dashboard is needed for them to appear though). Set `yes` for a chart instead of `auto` to enable it permanently. For example:
```
Comment (ilyam8) [inline collectors/cgroups.plugin/README.md]: :fox_face:

**Pull request merged**